File: test/export/cs/02-entrada-salida.cs
using System;
namespace PSeInt {
	class sin_titulo {
		static void Main(string[] args) {
			int a;
			double b;
			string c;
			bool d;
			a = int.Parse(Console.ReadLine());
			b = Double.Parse(Console.ReadLine());
			c = Console.ReadLine();
			d = Boolean.Parse(Console.ReadLine());
			Console.WriteLine(a);
			Console.WriteLine(b);
			Console.WriteLine(c);
			Console.WriteLine(d);
			a = int.Parse(Console.ReadLine());
			b = Double.Parse(Console.ReadLine());
			c = Console.ReadLine();
			d = Boolean.Parse(Console.ReadLine());
			Console.WriteLine(a.ToString(), b.ToString(), c, d.ToString());
			Console.Write("Esta linea no lleva enter al final");
		}
	}
}

File: test/export/js/02-entrada-salida.js
function sin_titulo() {
	var a = new Number();
	var b = new Number();
	var c = new String();
	var d = new Boolean();
	a = Number(prompt());
	b = Number(prompt());
	c = prompt();
	d = Boolean(prompt());
	document.write(a,'<BR/>');
	document.write(b,'<BR/>');
	document.write(c,'<BR/>');
	document.write(d,'<BR/>');
	a = Number(prompt());
	b = Number(prompt());
	c = prompt();
	d = Boolean(prompt());
	document.write(a,b,c,d,'<BR/>');
	document.write("Esta linea no lleva enter al final");
}
